_data/simple-icons.json

```
@@ -15138,7 +15138,7 @@
         },
         {
             "title": "Stripe",
-            "hex": "008CDD",
+            "hex": "7069FE",
             "source": "https://stripe.com/about/resources"
```

```suggestion
            "source": "https://stripe.com/newsroom/information"
```
This link now redirects here.